src/qtt/measurements/acquisition/lock_in_stimulus.py

```diff
@@ -0,0 +1,84 @@
+""" Provides the LockInStimulus"""
```

Comment:
Remove this comment? It does not add a lot of information

Reply:
done